src/odemis/driver/smarpod.py

```diff
@@ -412,8 +416,8 @@ def Move(self, pos):
         """
         Move to pose command. Non-blocking
         pos: (dict str -> float): axis name -> position
```

Comment:
I'd suggest to make that function has all the same parameters than the wrapped function `Smapod_Move()`. So the signature could be `Move(self, pos, hold_time=1, block=False)`. `hold_time` would be in s, with `float("inf")` mapped to infinite.

This has two advantages:
 * the "magical" constants are not magical and not constant anymore.
 * you don't reduce the functionality of this function.

Reply:
Should hold time be in seconds rather than ms? And as a float then?

Reply:
Yes, it should be a float.